src/components/Flyout/FlyoutFooter.tsx

```diff
@@ -14,7 +14,7 @@ export const FlyoutFooter: FC<FlyoutFooterProps> = ({ buttons, border = true })
     return (
         <div
             className={merge([
-                'tw-flex tw-gap-x-3 tw-justify-end tw-py-5 tw-px-8 tw-bg-white dark:tw-bg-black-95',
+                'tw-flex tw-gap-x-3 tw-justify-end tw-py-5 tw-px-8 tw-bg-white dark:tw-bg-black-95 tw-rounded-bl tw-rounded-br tw-border-l tw-border-b tw-border-r tw-border-line',
```

Comment:
Same here
`tw-border-l tw-border-b tw-border-r` could be also `tw-border tw-border-t-0`